lib/DependencyScan/ModuleDependencyCacheSerialization.cpp

```diff
@@ -474,10 +465,8 @@ bool ModuleDependenciesCacheDeserializer::readGraph(
 
       auto optionalOptionalImportStatementInfos =
           getOptionalImportStatementInfoArray(optionalImportsArrayID);
-      if (!optionalOptionalImportStatementInfos)
-        llvm::report_fatal_error(
-            "Bad direct Swift dependencies: no optional imports");
-      optionalImportStatements = optionalOptionalImportStatementInfos.value();
+      if (optionalOptionalImportStatementInfos.has_value())
```

Comment:
Use operator bool and operator *. There are no advantages to use has_value() and value() when you have proper checks.